app.js

```diff
@@ -0,0 +1,353 @@
+/*global alert ,confirm ,window ,setInterval ,setTimeout ,document ,clearInterval*/
+    //  Get user info
+(function getUserInfo() {
+    'use strict';
+    var url = window.location.href,
+        userName,
+        userAge,
+        greetDate,
+        userNamePlace,
+        getTimeNow;
+
+    //  if game page has no data(string) in url, keep moving.
+    //  Otherwise, declare user's name and age variables.
+    if ("string" === typeof url.split("?")[1]) {
+
+        userName = url.split("?")[1].split("=")[1].split("&")[0];
+        userAge = url.split("?")[1].split("=")[2];
+
+        //  Greeting New Guest
+        greetDate = document.getElementById('greeting-date');
+        userNamePlace = document.getElementById('user-name');
+        getTimeNow = new Date().getHours();
+
+        if (getTimeNow >= 5 && getTimeNow <= 11) {
```

Comment:
Use more descriptive variable names, ie `currentHour` instead of `getTimeNow`, so it is more readable by anyone reading the script.